### DISCUSSION TITLE
File missing error.

### DISCUSSION THREAD

**Discussion by shankhadevpadam** [laravel/framework#46020] [Q&A]
```php
vendor/composer/../doctrine/lexer/lib/Doctrine/Common/Lexer/AbstractLexer.php): Failed to open stream: No such file or directory",
```

**Comment by henzeb** [comment]
the answer.

can only be answered if we have more details then just an error: please, laravel version, php version, what you already tried. etc.

**Comment by crynobone** [comment]
Hey there,

We're closing this issue because it's inactive, already solved, old, or not relevant anymore. Feel free to open up a new issue if you're still experiencing this problem.
